server/webhook/merge_request.go

```diff
@@ -33,7 +38,18 @@ func (w *webhook) handleDMMergeRequest(event *gitlab.MergeEvent) ([]*HandleWebho
 		case actionReopen:
 			message = fmt.Sprintf("[%s](%s) reopen your merge request [%s!%v](%s)", senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername), event.ObjectAttributes.Target.PathWithNamespace, event.ObjectAttributes.IID, event.ObjectAttributes.URL)
 		case actionUpdate:
-			// TODO not enough check (opened/update) to say assigned to you...
+			toUsers = []string{}
```

Comment:
I think sending the "you're assigned" message to the author (not assignee) is not right. Because the author is not necessarily the assignee. What do you think about this?

Reply:
The author may want to know "the issue was assigned to user x", but if we're not sending that notification already then the PR's current state makes sense